agent/bench-scripts/tests/pbench-trafficgen/test-51.txt

```diff
@@ -0,0 +1,310 @@
++++ Running test-51 pbench-trafficgen
+tool trigger strings for start: "Starting binary-search" and for stop: "Finished binary-search" are now registered for tool group: "default"
+found IOMMU option, continuing
+found dpdk-tools, continuing
+Total number of benchmark iterations: 4
+cp: cannot stat '/var/tmp/pbench-test-bench/pbench-agent/tmp/trex_cfg.yaml': No such file or directory
+cp: cannot stat 'profile1': No such file or directory
+Starting iteration[1-profile1-0.002pct_drop] (1 of 4)
+test sample 1 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample1
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample1
+test sample 2 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample2
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample2
+test sample 3 of 3
+[pbench-tool-trigger] starting trigger processing of STDIN using tool group default triggers at /var/tmp/pbench-test-bench/pbench-agent/tools-v1-default/__trigger__
+[pbench-tool-trigger] start-trigger:"Starting binary-search" stop-trigger:"Finished binary-search"
+Starting binary-search
+[pbench-tool-trigger]pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample3
+Finished binary-search
+[pbench-tool-trigger]pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/trafficgen_test-51_tg:trex-profile_pf:profile1,profile2_ml:0.002,0.001_tt:bs_1900-01-01T00:00:00/1-default/sample3
+Iteration 1-profile1-0.002pct_drop complete (1 of 4), with 1 pass and 0 failures
+cp: cannot stat 'profile1': No such file or directory
```

Comment:
was it supposed to behave like this?

Reply:
No, I have added mock profile files to the test.